### PR TITLE
fix pgadmin container exit issue

### PR DESCRIPTION
**Pull request by tonyqus**
Reference: 
https://stackoverflow.com/questions/65036797/pgadmin-container-kills-postgres-when-using-docker-compose

I always got the following error from pgadmin container and then exit code 1

> 'test@user' does not appear to be a valid email address. Please reset the PGADMIN_DEFAULT_EMAIL environment variable and try again.

- I also added a specific volume for pgadmin - pgadmindata.
- 8080 port is occupied in most cases (same on my box). I changed it to 5433. 
- depends_on helps make sure pgadmin container starts after pg database container starts

### CODE REVIEW

**Comment by tonyqus** [comment]
while running pg_restore in the docker container, I always getting the following error even if I dropped schema public. 

> pg_restore: while PROCESSING TOC:
> pg_restore: from TOC entry 3; 2615 2200 SCHEMA public postgres
> pg_restore: error: could not execute query: ERROR:  schema "public" already exists
> Command was: CREATE SCHEMA public;

Looks it's better to specify the image tag of the postgresql. The behavior of different pg versions are different according to [this post](https://dba.stackexchange.com/questions/90258/pg-restore-archiver-db-could-not-execute-query-error-schema-public-alre). 

Question: which version of pg do you use to generate the dump file?

File: docker-compose.yml [hunk 1]
```diff
@@ -22,13 +22,18 @@ services:
...
-      - "8080:80"
+      - "5433:80"
+    volumes:
+      - pgadmindata:/var/lib/pgadmin
+    depends_on:
+      - postgres
     restart: "no"
 volumes:
   pgdata:
+  pgadmindata:
```
Comment (joaopgrassi) [inline docker-compose.yml]: Let's add an empty line here.

**Review by joaopgrassi — APPROVED** [review]
Just a minor nit. Thanks fixing this.

File: docker-compose.yml (hunk 1 above)
Comment (tonyqus) [reply docker-compose.yml]: done

**Comment by bruno-garcia** [comment]
> Question: which version of pg do you use to generate the dump file?

postgres 10 IIRC

File: docker-compose.yml [hunk 2]
```diff
@@ -22,13 +22,18 @@ services:
       - pgdata:/var/lib/postgresql/data
   pgadmin:
     environment:
-      PGADMIN_DEFAULT_EMAIL: test@user
+      PGADMIN_DEFAULT_EMAIL: test@user.com
```
Comment (bruno-garcia) [inline docker-compose.yml]: `test@user` is a valid email address, any reason to add `.com`? 

```suggestion
      PGADMIN_DEFAULT_EMAIL: test@user
```

**Review by bruno-garcia — COMMENTED** [review]
I have one nit, other than that lgbtm

File: docker-compose.yml (hunk 2 above)
Comment (bruno-garcia) [reply docker-compose.yml]: I'll revert this so it doesn't block the other changes

**Comment by bruno-garcia** [comment]
Not sure what's going on but CI isn't running

**Comment by bruno-garcia** [comment]
CI doesn't really verify this file so merging

**Pull request merged**